trajectory_executor/src/test/trajectory_executor_test_2.cpp

```diff
@@ -28,7 +28,7 @@ TEST_F(TrajectoryExecutorTestSuite, test_emit_multiple) {
 
     std::this_thread::sleep_for(std::chrono::milliseconds(2000));
 
-    ASSERT_LE(10, msg_count) << "Failed to receive whole trajectory from TrajectoryExecutor node.";
+    ASSERT_LE(msg_count, 10) << "Failed to receive whole trajectory from TrajectoryExecutor node.";
```

Comment:
I also think this was supposed to be the other way around as well, we want to check if the msg count is at least 10, so that we verify it's sending every point in the trajectory as it walks it's way through it.